requirements.txt

```diff
@@ -1,5 +1,7 @@
-aiogram==2.6.1
+python-telegram-bot==12.7
```

Comment:
Does it have full semver version? Please add a patch version as well, the same below

Reply:
fixed it